dists/go-ipfs/go-ipfs-dist

```diff
@@ -80,7 +80,7 @@ dist_version() {
 
   # generate dist.json file
   echo "---> generating dist.json"
-  ./mkdist.js "$version"
+  `which node` ./mkdist.js "$version"
```

Comment:
I know this doesn't solve the issues, but just want to pull it out, and make clear it's js file getting executed via node